core/rawdb/database.go

```diff
@@ -441,7 +475,7 @@ func InspectDatabase(db ethdb.Database, keyPrefix, keyStart []byte) error {
 	}
 	// Inspect append-only file store then.
 	ancientSizes := []*common.StorageSize{&ancientHeadersSize, &ancientBodiesSize, &ancientReceiptsSize, &ancientHashesSize, &ancientTdsSize}
-	for i, category := range []string{freezerHeaderTable, freezerBodiesTable, freezerReceiptTable, freezerHashTable, freezerDifficultyTable} {
```

Comment:
It looks to me like `freezerHeaderTable` changed to `chainFreezerHeaderTable`, but didn't go away entirely. I think we can leave our previous functionality, we'll just need to adjust it to account for this.